# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/canvas/crashtests/crashtests.list
# Commit: 78561892d142
# Full Hash: 78561892d142292d783d9adca93c72da7d45be87
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-10-10 21:33:39
# Description:
#   Bug 1857986 - add null check for mCanvasStyle r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D190546
# ==============================================================================

diff -r 0fd304fa622b -r 78561892d142 dom/canvas/crashtests/crashtests.list
--- a/dom/canvas/crashtests/crashtests.list	Tue Oct 10 12:49:38 2023 +0300
+++ b/dom/canvas/crashtests/crashtests.list	Tue Oct 10 09:26:22 2023 +0000
@@ -66,3 +66,4 @@
 pref(gfx.offscreencanvas.enabled,true) load 1771007-1.html
 pref(dom.webgpu.enabled,true) load 1816140.html
 load 1849704-1.html
+load 1857986.html
